src/test/resources/features/SearchDatasetByDescription.feature

```diff
@@ -0,0 +1,18 @@
+Feature: SearchDatasetByDescription
+  In order to list all datasets that match with a given description words
+  As a data user
+  I want to show the datasets that match with a description
+
+  Scenario: Show a dataset given a description
+    Given There is a dataset with description "dataset owned by owner" and owner "owner"
+    And There is 1 datasets registered
+    And I login as "owner" with password "password"
```

Comment:
In this case, as this implies a GET to the API, there is no need to be logged in. You can omit this from the scenarios or include additional scenarios where there is no authentication and it is shown that the system works just the same